packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -102,7 +102,9 @@ class RealAccountSignup extends React.Component {
                     title: () => localize('Add or manage account'),
                 },
                 {
-                    body: () => <StatusDialogContainer currency={this.props.state_value.currency} />,
+                    body: () => (
+                        <StatusDialogContainer currency={this.props.state_value.currency} onCancel={this.closeModal} />
```

Comment:
can we change `onCancel` prop to `closeModal` so that we don't have to add an extra prop(`onCancel`) in `status-dialog-container.jsx`?

Reply:
@balakrishna-binary you are right, `<StatusDialogContainer />` call `ui-store.closeRealAccountSignup,` on cancel or close, we already call it in `closeModal` of `real-account-signup.jsx`. I fixed the issue, as `<StatusDialogContainer />` has `onCancel` method so I use that for the purpose.